packages/custom-functions-metadata/Specification.md

```diff
@@ -11,6 +11,7 @@ The function parameter types may be provided using the [@param](#param) tag in J
 ## Tags
 * [@cancelable](#cancelable)
 * [@capturesCallingObject](#capturescallingobject)
+* [@customEnum](#customenum) _{type}_
```

Comment:
nit: Other lines don't use italic or curly brace around the parameter. It would be nice for them all to be consistent. I think italic without curly brace may be enough.

Reply:
Also, as commented below, if you want to denote optional parameters in square brackets and required parameters in curly braces, that is fine with me.

Reply:
Curly braces is not used to denote optional but as part of the type, for example `@customEnum {number}`. Wrapping type inside of curly braces seems to be a common practice, like custom function's `returns {number}` or TypeScript native `@enum {number}`: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#enum. 
BTW, I may need to remove this line because those tags are for functions.